src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest2.cs

```diff
@@ -1223,6 +1224,7 @@ public void BuildMultiDexApplication ()
 		}
 
 		[Test]
+		[Category ("DotNetIgnore")]
```

Comment:
Why is this `DotNetIgnore`?  I'm not sure how this is related to minSdkVerison.

Reply:
The asserts in this test are all centered around generation of multiple classes.dex files, which will no longer be produced now that the project has a minSdkVersion of 21.

Reply:
I removed the assert related to the 3rd classes.dex file as it is no longer produced now that the test is using default multidex support.